src/components/node-list/node-list.js

```diff
@@ -34,62 +35,57 @@ const NodeList = ({
       searchValue={searchValue}
     />
     <SplitPanel>
-      {
-        ({ isResizing, props: { container, panelA, panelB, handle } }) =>
-          <div 
-            className={classnames('pipeline-nodelist__split', {
-              'pipeline-nodelist__split--resizing': isResizing,
-            })}
-            {...container}
-          >
-            <div className="pipeline-nodelist__elements-panel" {...panelA}>
-              <Scrollbars
-                className="pipeline-nodelist-scrollbars"
-                style={{ width: 'auto' }}
-                autoHide
-                hideTracksWhenNotNeeded>
-                <NodeListGroups
-                  items={items}
-                  sections={sections.Elements}
-                  groups={groups}
+      {({ isResizing, props: { container, panelA, panelB, handle } }) => (
+        <div
+          className={classnames('pipeline-nodelist__split', {
+            'pipeline-nodelist__split--resizing': isResizing,
+          })}
+          {...container}>
+          <div className="pipeline-nodelist__elements-panel" {...panelA}>
+            <Scrollbars
+              className="pipeline-nodelist-scrollbars"
+              style={{ width: 'auto' }}
+              autoHide
+              hideTracksWhenNotNeeded>
+              <h2 className="pipeline-nodelist-section__title">Elements</h2>
```

Comment:
I think we can actually remove this section title now, though some of the designs I think still need updating @GabrielComymQB

Reply:
This has been taken out in the new implementation in the new PR